### PR TITLE
updated codes based on reviews

### PR DESCRIPTION
**Pull request by vinay-coder1**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,152 +1,307 @@
...
 <project xmlns="http://maven.apache.org/POM/4.0.0"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>com.xworkz</groupId>
-	<artifactId>xworkzMaillApp</artifactId>
-	<packaging>war</packaging>
+
+	<groupId>com.x-workz</groupId>
```
Comment (xworkzodc) [inline pom.xml]: com.xworkz

File: pom.xml [hunk 2]
```diff
@@ -1,152 +1,307 @@
...
+			<groupId>javax.servlet.jsp</groupId>
+			<artifactId>jsp-api</artifactId>
+			<version>${javax.servlet.jsp.version}</version>
+		</dependency>
 		<!-- https://mvnrepository.com/artifact/javax.servlet/jstl -->
 		<dependency>
 			<groupId>javax.servlet</groupId>
 			<artifactId>jstl</artifactId>
-			<version>1.2</version>
+			<version>${javax.servlet.version}</version>
```
Comment (xworkzodc) [inline pom.xml]: change to javax.jstl.version

File: src/main/java/com/xworkz/controller/LoginController.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.ModelAndView;
+
+import com.xworkz.dto.LoginDTO;
+import com.xworkz.service.LoginControllerService;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+@Getter
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/controller/LoginController.java]: remove @Getter @Setter for Controller's

File: src/main/java/com/xworkz/service/SMSService.java [hunk 4]
```diff
@@ -6,55 +6,148 @@
...
-
 import org.json.JSONObject;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
 
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/service/SMSService.java]: remove @Getter and @Setter from all classes , except entity, dto ......we dont need in service, controller or dao's

File: src/main/java/com/xworkz/service/OkHttpClientService.java [hunk 5]
```diff
@@ -23,6 +23,11 @@
...
 import com.xworkz.dto.SendMailDTO;
 
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
 @Service
 @Configuration
 public class OkHttpClientService {
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/service/OkHttpClientService.java]: Add Interface for All services and Implement It...

File: src/main/java/com/xworkz/controller/MailController.java [hunk 6]
```diff
@@ -33,15 +40,15 @@
 	@Value("${replyTo}")
 	private String replyTo1;
 	@Value("${fromName}")
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/controller/MailController.java]: where are this properties used?

File: src/main/java/com/xworkz/controller/MailController.java [hunk 7]
```diff
@@ -33,15 +40,15 @@
...
-	private String fromName1;
+	private String fromName1;  
 	
 	
 	@Autowired
 	private XworkzService xworkzService;
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/controller/MailController.java]: Should be interface references...class cannot have another class reference.Write interface's for all services

File: src/main/java/com/xworkz/dao/LoginControllerDAOImpl.java [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+@Repository
+public class LoginControllerDAOImpl implements LoginControllerDAO {
+	
+	static Logger logger = LoggerFactory.getLogger(LoginControllerDAOImpl.class);
+
+	public LoginControllerDAOImpl() {
+		logger.info("{} Is Created...........",this.getClass().getSimpleName());	}
+	
+	public static Logger getLogger() {
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/dao/LoginControllerDAOImpl.java]: remove get and set for logger's in call classes

File: src/main/java/com/xworkz/dao/LoginControllerDAOImpl.java [hunk 9]
```diff
@@ -0,0 +1,30 @@
+package com.xworkz.dao;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class LoginControllerDAOImpl implements LoginControllerDAO {
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/dao/LoginControllerDAOImpl.java]: You should not have DAO for generating OTP, it should be done in services...read my comments properly

File: src/main/java/com/xworkz/dto/CampaignDefault.java [hunk 10]
```diff
@@ -15,45 +22,5 @@
 	public CampaignDefault() {
 		super();
 	}
-
-	public String getFrom_name() {
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/dto/CampaignDefault.java:19]: remove under_score from properties

File: src/main/java/com/xworkz/dto/LoginDTO.java [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/dto/LoginDTO.java]: @Remove @ToString , this wil internally print password

File: src/main/java/com/xworkz/dto/MailChimpCampaign.java [hunk 12]
```diff
@@ -28,125 +35,6 @@
 	MailChimpDeliveryStatus delivery_status;
 	List<Links> _links;
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/dto/MailChimpCampaign.java]: remove under_scores

File: src/main/java/com/xworkz/dto/MailChimpEcommerce.java [hunk 13]
```diff
@@ -2,41 +2,23 @@
...
+@ToString
+public class MailChimpEcommerce implements Serializable {
+
 	private static final long serialVersionUID = 1L;
-	
+
 	long total_orders;
 	long total_spent;
 	long total_revenue;
-	public long getTotal_orders() {
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/dto/MailChimpEcommerce.java:12]: remove under scores..

File: src/main/java/com/xworkz/service/LoginControllerServiceImpl.java [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+	private String toMailId;
+	@Value("${to.MailSubject}")
+	private String toMailSubject;
+
+	public LoginControllerServiceImpl() {
+		logger.info("{} Is Created...........", this.getClass().getSimpleName());
+	}
+
+	@Override
+	public boolean validateAndLogin(LoginDTO dto, ModelMap model, HttpServletRequest request) {
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/service/LoginControllerServiceImpl.java]: controller cannot have ModelMap and Request...

File: src/main/java/com/xworkz/service/OkHttpClientService.java [hunk 15]
```diff
@@ -267,13 +272,12 @@ public String getCompaignDetails() {
...
 	}
 
-	public String getAllMailChimpList(Integer msgType) {
-		logger.info("qwertyuiop asdfghjkl zxcvbnm {} {}",msgType);
+	public String getAllMailChimpList(Integer mailid) {
 		String a = "";
 		String rt = "";
 		String fn = "";
 		String htp = "";
-		if (msgType == 1) {
```
Comment (xworkzodc) [inline src/main/java/com/xworkz/service/OkHttpClientService.java:276]: use enum or constants instead of 1

**Review by xworkzodc — CHANGES_REQUESTED** [review]

File: src/main/java/com/xworkz/controller/LoginController.java (hunk 3 above)
Comment (vinay-coder1) [reply src/main/java/com/xworkz/controller/LoginController.java]: for spring dependency injection we need setter method?

File: src/main/java/com/xworkz/dto/MailChimpList.java [hunk 16]
```diff
@@ -3,41 +3,20 @@
...
-	}
-	public List<Links> get_links() {
-		return _links;
-	}
-	public void set_links(List<Links> _links) {
-		this._links = _links;
-	}
 
-	
+	List<MailChimpMailDetails> lists;
```
Comment (ombn) [inline src/main/java/com/xworkz/dto/MailChimpList.java]: Make properties as private

File: src/main/java/com/xworkz/dto/MailChimpMailDetails.java [hunk 17]
```diff
@@ -1,231 +1,44 @@
...
-import java.util.Arrays;
 import java.util.List;
 
+import lombok.Getter;
+import lombok.Setter;
 
+@Getter
+@Setter
 public class MailChimpMailDetails implements Serializable {
-
```
Comment (ombn) [inline src/main/java/com/xworkz/dto/MailChimpMailDetails.java:9]: Make properties as private in every class

File: src/main/java/com/xworkz/service/LoginControllerServiceImpl.java [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+
+@Getter
+@Setter
+@Service
+public class LoginControllerServiceImpl implements LoginControllerService {
+
+	private Logger logger = LoggerFactory.getLogger(LoginControllerServiceImpl.class);
+
+	public String temporaryPass;
+	public LocalTime mailsentTime;
```
Comment (ombn) [inline src/main/java/com/xworkz/service/LoginControllerServiceImpl.java]: properties should not be public, if public why you need getter and setter

File: src/main/java/com/xworkz/service/LoginControllerServiceImpl.java (hunk 14 above)
Comment (ombn) [reply src/main/java/com/xworkz/service/LoginControllerServiceImpl.java]: still not fixed

File: src/main/java/com/xworkz/service/LoginControllerServiceImpl.java (hunk 18 above)
Comment (ombn) [reply src/main/java/com/xworkz/service/LoginControllerServiceImpl.java]: temporaryPass, mailSentTime and loginTime should be method level

File: src/main/java/com/xworkz/service/LoginServiceImpl.java [hunk 19]
```diff
@@ -0,0 +1,100 @@
...
+	public boolean generateOTP(HttpServletRequest request) {
+		logger.info("invoked generateOTP in service...");
+
+		try {
+			String onetimepass = genarateRandomOTP();
+			 temporaryPass = onetimepass;
+			if (Objects.nonNull(onetimepass)) {
+				logger.info("onetimepass generated in service...");
+				String mailID = toMailId;
+				String subject = toMailSubject;
```
Comment (ombn) [inline src/main/java/com/xworkz/service/LoginServiceImpl.java]: Load from template(HTML file)

File: src/main/webapp/css/index.css [hunk 20]
```diff
@@ -9,7 +9,7 @@ body {
 }
 
 .backgroundImage {
-	background-image: url("/xworkzMaillApp/img/prism.png");
+	background-image: url("https://raw.githubusercontent.com/vinay-coder1/xworkz-bulk-mail-app/master/prism.png");
```
Comment (ombn) [inline src/main/webapp/css/index.css]: this should load from xworkzodc..repo

**Review by ombn — CHANGES_REQUESTED** [review]

File: src/main/webapp/css/index.css (hunk 20 above)
Comment (vinay-coder1) [reply src/main/webapp/css/index.css]: need to make the repo public to achieve this sir

File: src/main/java/com/xworkz/service/LoginControllerServiceImpl.java (hunk 18 above)
Comment (vinay-coder1) [reply src/main/java/com/xworkz/service/LoginControllerServiceImpl.java]: temporaryPass, mailSentTime and loginTime , I'm accessing from two different method so I couldn't able to declare in method level sir.

**Pull request closed without merging**